src/screens/Term/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { Paper } from '@material-ui/core';
+import styled from 'styled-components';
+
+export const StyledPaper = styled(Paper)`
+  background-color: white;
+  color: var(--black);
+  width: 70%;
+  margin: 8px 0px;
+  padding: 0px 16px;
+  box-shadow: 0 10px 1em var(--black), 2.5px 5px 0.75em var(--primary) !important;
+`;
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 8px;
+
+  .MuiTypography-h1 {
+    margin-top: 0.9rem;
+    font-size: 2.5rem;
+  }
+
+  .MuiTypography-h2 {
+    font-size: 1.25rem;
+    font-weight: bold;
+  }
+  .MuiTypography-h3 {
+    margin-top: 1em;
+    font-size: 1.25rem;
+  }
```

Comment:
Agora que a aplicação tá crescendo mais isso aqui pode zuar outras páginas, vamos mudar isso aqui. Se não me engano dá pra obter o mesmo resultado fazendo um `&.className`.

Mais informações: https://www.sipios.com/blog-tech/how-to-use-styled-components-with-material-ui-in-a-react-app